src/relay/ir/module.cc

```diff
@@ -96,6 +98,25 @@ void ModuleNode::Update(const Module& mod) {
   }
 }
 
+Expr ModuleNode::EntryPoint() {
+  return this->Lookup(this->entry_func);
+}
+
+Module ModuleNode::FromExpr(
+  const Expr& expr,
+  const tvm::Map<GlobalVar, Function>& global_funcs) {
```

Comment:
`global_funcs ` is not used. Do we want this module to include all the functions in `global_funcs` as well?